core/templates/dev/head/css/oppia.css

```diff
@@ -3441,13 +3441,12 @@ md-card.preview-conversation-skin-supplemental-card {
   border-radius: 0;
   color: #fff;
   font-family: "Capriola", "Roboto", Arial, sans-serif;
-  font-size: 18px;
+  font-size: 14px;
```

Comment:
It seems like a better solution might be to allow the button width to scale without changing the font size, if possible